submission-scripts/script-templates/magpie-alluxio

```diff
@@ -0,0 +1,143 @@
+############################################################################
+# Alluxio Core Configurations
+############################################################################
+
+# Should Alluxio be run
+#
+# Specify yes or no. Defaults to no.
+#
+export ALLUXIO_SETUP=no
+
+# Alluxio Version
+#
+export ALLUXIO_VERSION="ALLUXIOVERSIONDEFAULT"
+
+# Path to your Alluxio build/binaries
+#
+# This should be accessible on all nodes in your allocation. Typically
+# this is in an NFS mount.
+#
+export ALLUXIO_HOME="ALLUXIODIRPREFIX/alluxio-${ALLUXIO_VERSION}"
+
+# ALLUXIO_WORKER_ON_MASTER
+#
+# Most big data projects do not have a "slave" or "worker" that also
```

Comment:
per my comment about a possible Magpie 3.0 release timing for Alluxio, lets remove "slave" and just call it "worker" only.